### PR TITLE
fix(API): iOS SPM and mutations

### PR DESCRIPTION
**Pull request by lawmicha**
_Issue #, if available:_

_Description of changes:_
- Adds SPM steps under API category
- Adds reference to create and delete mutation in the section "create, update, delete data" (renamed from "create, update")

1. iOS Changes
image
image

2. Android
image

3. Flutter

image

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/lib/graphqlapi/fragments/ios/mutate-data.md [hunk 1]
```diff
@@ -54,6 +67,14 @@ func updateTodo() -> AnyCancellable {
 }
 ```
 
+To create data, replace the request with `.create`
```
Comment (royjit) [inline docs/lib/graphqlapi/fragments/ios/mutate-data.md:70]: Can we add similar to Android and Flutter? Or create a backlog issue to add these.

File: docs/lib/graphqlapi/mutate-data.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Create or update data
+title: Create, update, delete data
```
Comment (royjit) [inline docs/lib/graphqlapi/mutate-data.md:2]: This change might affect other platforms.

Comment (lawmicha) [reply docs/lib/graphqlapi/mutate-data.md:2]: updating android and flutter docs in this PR as well

File: docs/lib/graphqlapi/fragments/ios/mutate-data.md (hunk 1 above)
Comment (lawmicha) [reply docs/lib/graphqlapi/fragments/ios/mutate-data.md:70]: yup, adding those in into this PR

**Review by ragingsquirrel3 — APPROVED** [review]
mostly focused on flutter part, but LGTM. Thanks for updating this.

**Review by royjit — APPROVED** [review]

**Comment by lawmicha** [comment]
@aws-amplify/amplify-sa please review/merge at your convenience, thanks!

**Review by siegerts — APPROVED** [review]

**Pull request merged**